utils/flags.go

```diff
@@ -447,4 +452,9 @@ var (
 		Usage:   "determines number of workers",
 		Value:   4,
 	}
+	UseGethBlockProcessorFlag = cli.BoolFlag{
+		Name:  "use-geth-block-processor",
+		Usage: "When set, use go-ethereum/core instead of go-opera/evmcore when ApplyMessage()",
+		Value: false,
+	}
```

Comment:
I think this flag is unrelated to this PR and is added [here](https://github.com/Fantom-foundation/Aida/pull/1171)

Reply:
Thanks for the catch, removed the stray commit.